src/steps/safeOnboarding/fundsToSafe.tsx

```diff
@@ -41,24 +40,18 @@ const FundsToSafe = ({
   set_step,
 }: FundsToSafeProps) => {
   const selectedSafeAddress = useAppSelector((store) => store.safe.selectedSafeAddress);
+  const walletBalance = useAppSelector((store) => store.web3.balance);
```

Comment:
Same comment as above, if one of the values comprehended inside the `walletBalance`, would it re-render the whole component?

Reply:
Yes, good point but it will be a good thing to work on in a cople of months. 2